packages/react/src/components/Icon/Icon.tsx

```diff
@@ -45,50 +67,97 @@ export interface IconProps extends UIComponentProps, ColorComponentProps {
   xSpacing?: IconXSpacing
 }
 
-class Icon extends UIComponent<WithAsProp<IconProps>, any> {
-  static create: ShorthandFactory<IconProps>
+const Icon: React.FC<WithAsProp<IconProps>> & FluentComponentStaticProps = props => {
+  const {
```

Comment:
Let's extract the props after the `setStart()`

Reply:
Done 👍